net/ser2net/files/etc/config/ser2net

```diff
@@ -0,0 +1,7 @@
+config 'proxy'
+        option 'tcpport' '8000'
+        option 'state' 'raw'
```

Comment:
Yes, I know "upstream" calls this state... but please change it to "protocol".

Reply:
Why? when I saw protocol in this later, I thought it was an error, mixingin tcp/udp options or v4/v6, both which go in the "port" section of the config string.